### ISSUE TITLE
Spiked to PlayerCell doesn't work?

### ISSUE THREAD

**Issue by makmm** [OgarProject/Ogar#283]
´PlayerCell.prototype.sendUpdate = function() {this.spiked = 1;};´
thats the code that i added to ´customGamemode.prototype.onServerInit´ and now the player can't move. help!! :/

**Comment by vram4** [comment]
Yes, because the sendUpdate function determines whether or not movement data is sent to the client, if you override it that way, then it'll break the server.

**Comment by makmm** [comment]
how i fix it? ;(

**Comment by Agarian** [comment]
it does work. you may have done something wrong?

**Comment by Agarian** [comment]
the correct line of code is in folder "entity" then "cell.js" find this line of code: this.spiked = 0;

**Comment by makmm** [comment]
yes, i know that but i want to make a gamemode that makes the PlayerCell spiked.

**Comment by Agarian** [comment]
that is technically impossible. doing so causes the server to become very unstable.

**Comment by Agarian** [comment]
i cannot help you there.

**Comment by NatsuTheGreat** [comment]
I can help you. 

In playerCell.js you have the this:

function PlayerCell() {
    Cell.apply(this, Array.prototype.slice.call(arguments));

    this.cellType = 0;
    this.recombineTicks = 0; // Ticks until the cell can recombine with other cells 
    this.ignoreCollision = false; // This is used by player cells so that they dont cause any problems when splitting
}

After this.ignoreCollision add:
this.spiked = 1;

And then all the player cells will be spiked, but not food cells, or any other.

You'll have viruses and players spiked

If you have any questions, ask me.

**Comment by AgarioCheats** [comment]
@NatsuTheGreat Nice find, Natsu. That's about the same results i try to experiment with, without too much coding experience.

**Comment by NatsuTheGreat** [comment]
@AgarioCheats That's nice. How's your server by the way, we all want to see how you did on your splitting. xD

**Comment by Agarian** [comment]
@AgarioCheats yeah because i still couldn't manage to get the code into wherever it belonged. so i put it in gameserver.js by other *split* functions but when i did if split.ignoreCollision was set to true then cells never collide. If its set to false then cells always collide. Removing the line gets me back to where i started. cell.ignoreCollision doesn't seem to affect anything. I must know exactly where the code goes. because the console is picky about it.

**Comment by Agarian** [comment]
@AgarioCheats y u never reply to me?
![download](https://cloud.githubusercontent.com/assets/15255724/12066402/91e76264-afb5-11e5-880d-b886527ca676.jpg)

**Comment by Andrews54757** [comment]
My fork at https://github.com/AJS-development/Ogar-unlimited has a feature where you can enable spikes in gameServer.in

**Comment by Agarian** [comment]
It gave an error:
Cannot read the property 'config' of undefined

**Comment by Andrews54757** [comment]
oops

**Comment by Andrews54757** [comment]
il fix it

**Comment by Andrews54757** [comment]
(Me and my messy code)

**Comment by Andrews54757** [comment]
I fixed the issue and it should work now at https://github.com/AJS-development/Ogar-unlimited
